### ISSUE TITLE
Langkawi mistakenly included in Perlis (should be Kedah) -- Malaysia

### ISSUE THREAD

**Issue by MichaelChirico** [nvkelso/natural-earth-vector#277]
https://en.wikipedia.org/wiki/Langkawi

> Langkawi, officially known as Langkawi, the Jewel of Kedah (Malay: Langkawi Permata Kedah), is a district and an archipelago of 99 islands (+ 5 small islands visible only at low tide) in the Andaman Sea some 30 km off the mainland coast of northwestern Malaysia. The islands are a part of the state of Kedah, which is adjacent to the Thai border.

<img width="735" alt="screen shot 2018-10-14 at 11 11 16 pm" src="https://user-images.githubusercontent.com/7606389/46918605-79fff980-d006-11e8-8f19-bedca10bcc85.png">

**Comment by nvkelso** [comment]
Thanks for these reports! I suspect this is a combination of topology geometry errors preventing MapShaper from assembling the right output, or the label points don’t have good attributes for filter. See the Virginia issue that’s recently commented on.

**Comment by MichaelChirico** [comment]
HTH... it's a huge map so I understand if it's hard to keep track of every nook and cranny of the planet... just so happens I'm using this for a project & came across a bunch of inconsistencies in SEA. 

This may be different w this vs. the VA issue...

 1. That issue is about a polygon clipping; this one here is about mistaken identity (all the polygons including the island of Langkawi are present)
 2. The other two issues (#275, #274) are closer to the VA issue, but still strike me as odd because in both cases there seems to be a specific carve-out of a contiguous city from the same land mass (the neighboring islands I mention in both issues fit better the pattern of possible polygon filtering).

Not sure I understand too well the make process for this repo or I might try my hand at fixing this issue in particular (just need to reassign the island polygon)

**Comment by MichaelChirico** [comment]
Closed the other issues, but this one is still a problem in current master:

![screen shot 2018-10-16 at 6 20 59 pm](https://user-images.githubusercontent.com/7606389/47009928-3de8a800-d170-11e8-8e00-5e136f644a0d.png)

```
library(rgdal)
x = readOGR('.', 'ne_10m_admin_1_states_provinces')

plot(p <- x[which(x$name_en == 'Perlis'), ], lwd = 3)
text(coordinates(p), labels = p$name_vi)
plot(x, add = TRUE, border = 'gray')
```

**Comment by MrGawsome** [comment]
I can confirm that Langkawi is still mistakingly part of the state of Perlis when it should be part of state of Kedah. This issue is on both the ne_10m_admin_1_state_provinces and ne_10m_admin_1_state_provinces_lakes shapefiles and doesn’t seem to be present on other shapefiles (for example, these states aren’t on the 50m shapefiles). 

Langkawi should be split from Perlis (ne_id 1159310067) and merged with Kedah (ne_id 1159310065) on both of the above shapefiles.

With some online research, I can confirm that Langkawi should be part of Kedah via various government and other sources. See below:

Malaysian government website “MyGeoName,” with a screenshot of the Kedah state below: http://mygeoname.mygeoportal.gov.my/index.jsp
![Screen Shot 2021-03-29 at 5 15 26 PM](https://user-images.githubusercontent.com/78829897/112915755-e32f3680-90b3-11eb-8f98-4c62663761ee.png)
Malaysian gov map which includes Langkawi as part of Kedah: https://www.jupem.gov.my/jupem18a/assets/uploads/images/contents/20190530085409-MY90008R.jpg
(The above map was taken from the “Themantic Map” section at https://www.jupem.gov.my/page/thematic-map)
Nationsonline.org map: https://www.nationsonline.org/oneworld/map/malaysia_map.htm
Indiatimes article which mentions that Langkawi is the “Jewel of Kedah” https://timesofindia.indiatimes.com/travel/things-to-do/paithani-and-himroo-fabric-shopping/gs50836194.cms
Kedah wiki: https://en.wikipedia.org/wiki/Kedah

**Comment by nvkelso** [comment]
Process detailed in https://github.com/nvkelso/natural-earth-vector/issues/396#issuecomment-789428141.

**Comment by nvkelso** [comment]
Fixed via #523 and #524.
